### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Tratcher**
Adding some build notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,7 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/aspnet
 ASP.NET Templates provide project templates which are used in .NET Core for creating ASP.NET Core applications.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+## Building Templates
+- Running build.cmd in this repo requires NPM which can be installed from https://nodejs.org/en/.
+- The ASP.NET localhost development certificate must also be installed and trusted or else you'll get a test error "Certificate error: Navigation blocked".
```
Comment (Eilon) [inline README.md]: How do I do this, or check that it's done?

Comment (Tratcher) [reply README.md]: Ask @javiercn, he's still writing those instructions 😄

Comment (javiercn) [reply README.md]: The tests put the cert in place for you automatically, there are a couple of options:
* Setup ASPNETCORE_BROWSER_AUTOMATION_FIREFOX=true to run the automation tests with Firefox.
* Trust the certificate on your machine:
  * Manually copying the localhost certificate on the personal current user store to the trusted root certification authorities user store
  * Running the dev-certs tool on your machine (one time operation) to setup the certificate to trusted.
    * Clone the dotnet tools repo, build and run `dotnet <<path-to-dev-certs>>.dll https -t` (accept when prompted)
  * Install the tool as a global tool using `dotnet tools install dev-certs --nugetconfig <<nugetconfigfilewithourmygetfeed>>`
    * Run dotnet dev-certs https -t

Comment (Tratcher) [reply README.md]: Nice. Put those instructions in the dev certs readme and I'll link to them from here.

File: README.md [hunk 2]
```diff
@@ -10,3 +10,4 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 ## Building Templates
 - Running build.cmd in this repo requires NPM which can be installed from https://nodejs.org/en/.
 - The ASP.NET localhost development certificate must also be installed and trusted or else you'll get a test error "Certificate error: Navigation blocked".
+- build.cmd (or build /t:compile) will produce nuget packages for each class of template in the artifacts directory. These can be installed via `dotnet new -i {nugetpackage path}`
```
Comment (javiercn) [inline README.md]: You also need to get the packages on your package cache or else when you do dotnet new restore will fail. The easiest way to get them to run is by letting the build run at least 1 test.

File: README.md [hunk 3]
```diff
@@ -10,4 +10,5 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 ## Building Templates
 - Running build.cmd in this repo requires NPM which can be installed from https://nodejs.org/en/.
 - The ASP.NET localhost development certificate must also be installed and trusted or else you'll get a test error "Certificate error: Navigation blocked".
-- build.cmd (or build /t:package to avoid tests) will produce nuget packages for each class of template in the artifacts directory. These can be installed via `dotnet new -i {nugetpackage path}`
+- `build.cmd` (or `build /t:package` to avoid tests) will produce nuget packages for each class of template in the artifacts directory. These can be installed via `dotnet new -i {nugetpackage path}`
```
Comment (Eilon) [inline README.md]: `nuget` --> `NuGet` when used in prose.

File: README.md (hunk 1 above)
Comment (Eilon) [reply README.md]: Are we just waiting for the dev certs docs? @javiercn , is that done, so that it can be linked from here?

Comment (Tratcher) [reply README.md]: Yes. @javiercn declined to document the current process because he said the real process will be available in a few weeks.

Comment (Eilon) [reply README.md]: Ok

**Pull request merged**